integration_tests/20_docker_additional_hosts.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+my_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+source "$my_dir/header.sh" "$my_dir/20_env.toml"
+
+docker build ./plans/additional_hosts/http_server -t http_server
+docker run -it -d -p 8080:8080 --name http_server http_server
+docker network connect testground-control http_server
```

Comment:
- I believe we need to call `testground healthcheck --runner local:docker --fix` first (we might run this on a fresh session without the control network).
- Could you use something like https://hub.docker.com/r/hashicorp/http-echo/ ? it's 4mo instead of go's 300.

Reply:
Good call on both points!
- [3668f03](https://github.com/testground/testground/pull/1481/commits/3668f0322aac37b167f58d16c1a60add9295e7d4) moves the healthcheck toward the start of the integration test script.
- [8137ce1](https://github.com/testground/testground/pull/1481/commits/8137ce14812647639d0025e82b88c7377ff9bf37) removes the custom http server and replaces it with hashicorp/http-echo.